src/main/java/frc/robot/constants/Constants.java

```diff
@@ -62,6 +65,54 @@ public static class Shooter {
             new Rotation3d(0.0, 0.0, Math.toRadians(180.0)));
   }
 
+  public static class Vision {
+    public enum CameraType {
+      AprilTagCamera
+    }
```

Comment:
Camera Type is no longer needed because we aren't doing ML-based cameras. Remove this